sdk/python/feast/on_demand_feature_view.py

```diff
@@ -64,8 +70,13 @@ def __init__(
         self,
         name: str,
         features: List[Feature],
-        inputs: Dict[str, Union[FeatureView, FeatureViewProjection, RequestDataSource]],
         udf: MethodType,
+        inputs: Optional[
+            Dict[str, Union[FeatureView, FeatureViewProjection, RequestDataSource]]
+        ] = None,
```

Comment:
I think making this a `kwarg` from an `arg` is a backwards incompatibile change. e.g.
```
ODFV(
  "my name",
  [...],
  {"input1": some_feature_view}
)
```

will no longer be valid. Are you thinking that this is okay because it's experimental?

Reply:
reordering parameters so that this doesn't break